about.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link href="styles-about.css" rel="stylesheet" />
+    <title>About me.</title>
+  </head>
+
+  <body>
```

Comment:
there's a small border around the body which can be removed to make the page look a little more seamless

<img width="1408" alt="image" src="https://user-images.githubusercontent.com/40119924/189636738-da52002f-9465-4965-9c6b-471e4b71cd0e.png">